src/sass/app_2/modules/trading.scss

```diff
@@ -309,6 +284,9 @@
         height: calc(100% + 2px);
         left: -2px;
     }
+    &__popover-bubble {
+        width: 220px;
```

Comment:
can we make the width according to the content ?, or maybe pass it as props

Reply:
It's a part of the design though and I don't think we should pass styles as props when we already have stylesheets. The reason we use margin prop is because the position is calculated in JS